projects/uitdatabank/docs/entry-api/requirements-before-going-live.md

```diff
@@ -0,0 +1,55 @@
+---
+stoplight-id: 73ef6e15ee3ec
+---
+
+# Requirements before going live
+
+For the integration with the UiTdatabank Entry API, there is a flow in which you first obtain access to our the test environment. 
+Access to the production environment is given after your integration on test environment has been successful validated by publiq vzw.
+
+On this page we describe how this flow works and how you can ensure that you gain access to the production environment as quickly as possible!
+
+> Make sure you read this guide before you start developing on our test environment!
+
+## Step 1: start on the test environment
+
+You can apply for free test credentials at [Projectaanvraag](https://projectaanvraag.uitdatabank.be). 
+
+Once you've obtained your personal test credentials you can start with the development of your integration with the UiTdatabank Entry API on our test environment.
+
+## Step 2: validation by publiq vzw
+
+As soon as you have finished developing your integration with the Entry API, you send a minimum of 5 of events to our test environment. You contact us via `vragen@publiq.be` with the email subject "content check". In the email you give us the identifiers of the created test events. We will then validate your integration as soon as possible. 
+
+In this validation process we assess the integration in two manners:
+1.  Is the **created data** qualitative, relevant and valuable for public agendas such as www.uitinvlaanderen.be?
+2.  Is the **integration made robust**, e.g. is it possible to update and delete content through the integration?
+
+It is important that you go through the checklist below in advance and validate your integration against it. By doing this in advance, you ensure a smooth transition to the production environment!
+
+### Checklist for a succesfull integration
+#### Mandatory properties
+-   `name`: only use plain text 
+-   `terms`: exactly one term of the domain `eventtype` is required. We strongly encourage to also include one term of the domain `theme` for events (see our [guide on taxonomy terms](../docs/terms.md) for more info)
```

Comment:
```suggestion
-   `terms`: exactly one term of the domain `eventtype` is required. We strongly encourage to also include one term of the domain `theme` for events (see our [guide on taxonomy terms](../../docs/terms.md) for more info)
```

The linter reports this as a dead link. The path needs an extra `../` because this specific file is inside a subdirectory (`entry-api`)